tests/unit/test_github.py

```diff
@@ -65,17 +64,11 @@ def test_create_gist(self):
             'example.py': {'content': '# example contents'}
         })
 
+        get_gist_example_data = create_example_data_helper('create_gist_example')
+
         self.post_called_with(
```

Comment:
@sigmavirus24 I was thinking the `json` directory would only contain example data from the API docs. What are your thoughts on the change here? Serializing the dictionary arguments for the assertion?